mimic/test/behavior_tests.py

```diff
@@ -0,0 +1,213 @@
+"""
+Automatically generate tests for behavior registration/deletion APIs.
+Also contains helper functions for specific behavior testing.
+"""
+import json
+
+from uuid import UUID, uuid4
+
+from six import text_type, string_types
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from zope.interface import Attribute, Interface
+
+from mimic.test.helpers import json_request, request_with_content
+
+
+class IBehaviorAPITestHelper(Interface):
+    """
+    Helper class that provides some setup and assertion methods that tests
```

Comment:
I think we could get rid of "Helper class that provides some setup and assertion methods that" and instead simply say:

> "Set-up and assertions required for testing behavior CRUD."

It might be good to have a reference to `make_behavior_tests` here, because what this docstring is *really* trying to say is "that thing you need for `make_behavior_tests` and not really for anything else".